net/src/codec/buffer.rs

```diff
@@ -1,88 +1,56 @@
+use bytes::{Buf, BytesMut};
+
 pub type ByteBuffer = Buffer;
 
-#[derive(Debug)]
+#[derive(Debug, Default)]
 pub struct Buffer {
```

Comment:
At some point it might be worth breaking the API and remove this wrapper which does nothing anymore